ST_indivs.c

```diff
@@ -190,22 +194,22 @@ Bool indiv_Kill_Partial( MortalityType code,
   return( result);
 }
 
+/**
+ * \brief Kill a portion of an individual plant.
+ * 
+ * Remove individual proportionally and adjust relative size.
+ * Also keep up with survivorship data.
+ * 
+ * \param ndv A pointer to the individual.
+ * \param killType The MortalityType code. This parameter is currently unused.
```

Comment:
I agree. It looks like at one point the field was used, but it has been deprecated slowly by misuse or disuse. Each individual has a `killedby` field but instead of using the MortalityType enumerator is it set to hard coded values like `indv->killedby = 1` so there is no guarantee that the values make sense right now. This inconsistency could lead to issues like #336 if the enum and the hard-coded values ever come out of sync.

Reply:
@chaukap still needs documentation as an issue.